pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -765,7 +780,7 @@ func (l *LibvirtDomainManager) PrepareMigrationTarget(vmi *v1.VirtualMachineInst
 	for _, port := range migrationPortsRange {
 		// Prepare the direct migration proxy
 		key := migrationproxy.ConstructProxyKey(string(vmi.UID), port)
-		curDirectAddress := fmt.Sprintf("%s:%d", "127.0.0.1", port)
+		curDirectAddress := fmt.Sprintf("%s:%d", loopbackAddress(), port)
```

Comment:
The ipv6 pr also introduced getLoopbackAddress(..) method. Maybe it can be moved to Common and reused.

Reply:
cant see it now on 3112  ? once it there i will look on it if its ok